src/applications/static-pages/sass/modules/_m-facilities.scss

```diff
@@ -40,5 +40,162 @@ $duo-tone-lighten: #000000;
 }
 
 .facility-satisfaction-tile {
-  width: 147px;
+  width: 50%;
+
+  @media all and (min-width: 768px) {
+    width: 147px;
+  }
+}
+
+.basic-facilities-list {
+  .facility {
+    margin-bottom: 32px;
+
+    @media all and (min-width: 768px) {
+      margin-bottom: 40px;
+    }
+
+    &:last-child {
+      margin-bottom: 24px;
+    }
+  }
+}
+
+.region-img,
+.news-img {
+  width: 100%;
+  height: 165px;
+  object-fit: cover;
+
+  @media all and (min-width: 481px) {
+    height: 180px;
+    width: 342px;
+  }
+
+  @media all and (min-width: 768px) {
+    height: initial;
+    width: initial;
+    object-fit: initial;
+  }
+}
+
+.facility-img {
+  width: 100%;
+  height: 164px;
+  object-fit: cover;
+
+  @media all and (min-width: 481px) {
+    height: 180px;
+    width: 342px;
+  }
+
+  @media all and (min-width: 768px) {
+    height: initial;
+    width: initial;
+    object-fit: initial;
+  }
+}
+
+.news-img {
+  @media all and (min-width: 481px) {
+    height: 180px;
+    width: 100%;
+  }
+
+  @media all and (min-width: 768px) {
+    height: initial;
+    width: initial;
+  }
+}
+
+.stories-list {
+  .heading-level-3 {
```

Comment:
Rather than overriding this class, could you add another class to the markup and style that? Overriding these common classes can be confusing later on.